### DISCUSSION TITLE
API changes for 2.0

### DISCUSSION THREAD

**Discussion by sdeastham** [MIT-LAE/APCEMM#41] [Ideas]
In APCEMM 2.0, we're going to change how the model is run and output is generated. This is a place to collect thoughts on changes we want/plan/hope to make to the model which will change how you interact with it! (Looking at you, @lrobion )

* input.yaml will have a large number of unused inputs deleted
* Default options will be used when not specified in input.yaml
* Output data will be stored with dimensions of [time, pressure], instead of [altitude, time]

**Comment by Calebsakhtar** [comment]
Suggestion from @sdeastham :
- Convert the ice number threshold and the contrail threshold to physical quantities that are easy to understand.

A finer calibration for these two parameters together is needed. A superficial calibration exercise has already been performed in PR https://github.com/MIT-LAE/APCEMM/pull/45

**Comment by Calebsakhtar** [comment]
I have noticed that if the lifetime does not match the saving frequency, APCEMM will only output data on the LCM of the lifetime and the saving frequency.

From a user point of view, either interpolating (not ideal), or getting the closest output time above or below the selected time would be nice.
